### DISCUSSION TITLE
Remote Managment, Network Management

### DISCUSSION THREAD

**Discussion by gwbridge** [meshtastic/Discussions#149] [General]
Coming from the WISP and IP networking space, I'm looking for a way to remotely manage all of the nodes on the network to remotely change configurations or push out firmware updates. I can't find this feature in the documentation. Is it possible?

Additionally, are there any base stations or handhelds that support dual loading and auto roll back if there is a firmware update failure?

**Comment by garthvh** [comment]
You can remotely administer nodes, firmware updates over Lora is not possible.

**Reply by gwbridge** [reply]
Thanks Garth! Is that a limitation in MeshTastic? Traditional LoRaWAN absolutely has remote management and firmware management.

**Reply by gwbridge** [reply]
Here is an example of a company that does it: https://loriot.io/

**Reply by madeofstown** [reply]
> Thanks Garth! Is that a limitation in MeshTastic? Traditional LoRaWAN absolutely has remote management and firmware management.

It's been asked about before but I believe the main takeaway is that the storage space on the nodes are too small to store the running firmware and the update firmware. Also, data transfer over Meshtastic is painfully slow and unreliable without a good direct link.
